test/common/run-tests

```diff
@@ -91,7 +91,6 @@ class Test:
         print_tap = not opts.list
         affected = any([self.command[0].endswith(t) for t in affected_tests])
         retry_reason = ""
-        skip_reason = ""
```

Comment:
`skip_reason` is being used again in line 124. This is plausibly the cause of SKIPed tests now being broken?